### PR TITLE
Fix CoffeeScript code for Facebook

### PR DESCRIPTION
**Pull request by manuelmeurer**
The current solution doesn't work for me. The main problem is that in `saveFacebookRoot` `$('#fb-root').detach()` is called even if `#fb-root'` did not (yet) exist, and in `restoreFacebookRoot` `#fb-root` is replaced with `fb_root` even if `fb_root` is empty. I added conditions to both functions for this.
I also fixed the variable names from snake_case to camelCase.
Also, I removed the `channelUrl` param, [which is deprecated](https://developers.facebook.com/blog/post/2013/11/21/platform-updates--new-design-for-follow-button-and-like-box--ios-sdk-improvements-and-more/), and [fixed the SDK load path from `all.js` to `sdk.js`](https://developers.facebook.com/docs/apps/upgrading)

### CODE REVIEW

**Comment by reed** [comment]
Thanks for submitting.  I merged this via the command line, so I'm not sure why it's not showing as merged.  I guess I'll just have to close this manually.

**Comment by manuelmeurer** [comment]
:+1:
